app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -696,13 +732,13 @@ class TabTrayView(
             // We offset the tab index by the number of items in the other adapters.
             // We add the offset, because the layoutManager is initialized with `reverseLayout`.
             val recyclerViewIndex = selectedBrowserTabIndex +
-                collectionsButtonAdapter.itemCount +
-                syncedTabsController.adapter.itemCount
+                    collectionsButtonAdapter.itemCount +
+                    syncedTabsController.adapter.itemCount
```

Comment:
It's wild that my IDE also reformats it back to the former, but I have all the other formatting changes in this file.